cmd/gcbuilder/cmd/root.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
this should be 2020? or 2019 is ok?

Reply:
@cpanato -- This is a migration of an older file from k/test-infra, so I wanted to keep the copyright dates intact.

Reply:
thanks!